Jenkinsfile

```diff
@@ -87,6 +92,27 @@
     }
 }
 
+def startLogCatCollector() {
+    sh '''rm -f logcat.txt
+    adb logcat -c
+    adb logcat > "logcat.txt" &
+    '''
+}
+
+def stopLogCatCollector(String pid, boolean archiveLog) {
+    sh '''jobs
+       kill %1'
+       '''
+    if (archiveLog) {
+        zip([
+            'zipFile': 'logcat.zip',
+            'archive': true,
```

Comment:
You mean `'archive' : 'archivelog',` ? 
This https://jenkins.io/doc/pipeline/steps/pipeline-utility-steps/ says it is a boolean?